src/windows/include/err.h

```diff
@@ -45,7 +45,7 @@
  * err - windows implementation of unix err function
  */
 __declspec(noreturn) static void
-err(int eval, const char *fmt, ...)
+err(int eval, const char *fmt, ...) __attribute__ ((format (printf, 2, 3)))
```

Comment:
This is gcc-specific attribute. Adding it to windows compatibility file does not make much sense.